draft-ietf-masque-h3-datagram.md

```diff
@@ -196,7 +196,7 @@ client-initiated bidirectional streams, and those have stream IDs that are
 divisible by four). The largest legal QUIC stream ID value is 2<sup>62</sup>-1,
 so the largest legal value of Quarter Stream ID is 2<sup>62</sup>-1 / 4.
 Receipt of a frame that includes a larger value MUST be treated as a connection
-error of type TBD.
+error of type H3_DATAGRAM_ERROR.
```

Comment:
I think we'll also want to use H3_DATAGRAM_ERROR to replace any use of H3_GENERAL_PROTOCOL_ERROR in this section